tasks/src/zero-shot-classification/about.md

```diff
@@ -1,20 +1,32 @@
-## Use Cases
+# Zero Shot Classification
 
-### Sentiment Analysis
+## About the Task
 
-Zero-shot learning is a variant of machine learning, where a model is trained on a set of data that does not include the target labels, and is able to predict the labels for new data that does not include the target labels. This can be used for sentiment classification by training a model on a dataset of texts that are not labeled for sentiment. The model can then be used to predict the sentiment of new texts that are not labeled for sentiment.
+Zero Shot Classification is a name for prompting a large language model with a task in natural language.  This method, which leverages a pre-trained language model, can be thought of as an instance of [transfer learning](https://www.youtube.com/watch?v=BqqfQnyjmgg) which generally refers to using a model trained for one task in a different application than what it was originally trained for. 
```

Comment:
I think there should be an emphasis of the fact that model hasn't seen the classes previously, as zero shot models are particularly good for when there's no labeled data.